From af05c7d20f000d6bd5cb3a012e1b3aea04cca800 Mon Sep 17 00:00:00 2001
From: Lindsey Gray <lindsey.gray@gmail.com>
Date: Tue, 10 Jun 2025 15:05:42 -0500
Subject: [PATCH] patches to use conda dependencies

---
 CMakeLists.txt | 9 +++++++--
 1 file changed, 7 insertions(+), 2 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index eb56203..e273837 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -13,6 +13,7 @@ option(SKHEPFJ_USE_INSTALLED_FASTJET          "Use an existing installed version
 option(SKHEPFJ_USE_INSTALLED_FASTJET_CONTRIB  "Use an existing installed version of fastjet-contrib" OFF)
 option(SKHEPFJ_PATCH_FJCORE                   "Patch fastjet-contrib with CMAKE_INSTALL_LIBDIR use"  ON)
 option(SKHEPFJ_PATCH_FJCONTRIB                "Patch fastjet-contrib with cmake build"               ON)
+option(SKHEPFJ_FASTJET_CXX_SWIG               "Create a _swig.py that just imports fastjet_cxx"      OFF)
 
 set(PKG_INSTALL "fastjet") ## python package name
 
@@ -44,6 +45,10 @@ else()
   add_subdirectory(${CMAKE_SOURCE_DIR}/extern/fastjet-contrib)
 endif()
 
+if (SKHEPFJ_FASTJET_CXX_SWIG)
+  file(WRITE ${CMAKE_INSTALL_PREFIX}/_swig.py "from fastjet_cxx import *\n")
+endif()
+
 pybind11_add_module(_ext MODULE src/_ext.cpp)
 target_link_libraries(_ext
                       PRIVATE
@@ -57,9 +62,9 @@ target_include_directories(_ext PRIVATE ${CMAKE_BINARY_DIR}/include)
 #set_target_properties(_ext PROPERTIES BUILD_WITH_INSTALL_RPATH TRUE)
 #set_target_properties(_ext PROPERTIES INSTALL_RPATH_USE_LINK_PATH TRUE)
 if (${CMAKE_SYSTEM_NAME} MATCHES "Darwin")
-  set_target_properties(_ext PROPERTIES INSTALL_RPATH "@loader_path/${CMAKE_INSTALL_LIBDIR}")
+  set_target_properties(_ext PROPERTIES INSTALL_RPATH "@loader_path/${CMAKE_INSTALL_LIBDIR};@loader_path/../../../../${CMAKE_INSTALL_LIBDIR}")
 elseif(NOT MSVC)
-  set_target_properties(_ext PROPERTIES INSTALL_RPATH "$ORIGIN/${CMAKE_INSTALL_LIBDIR}")
+  set_target_properties(_ext PROPERTIES INSTALL_RPATH "$ORIGIN/${CMAKE_INSTALL_LIBDIR}:$ORIGIN/../../../../${CMAKE_INSTALL_LIBDIR}")
 endif()
 
 if (MSVC AND FASTJET_ENABLE_CGAL)
-- 
2.39.5 (Apple Git-154)
